registry.json

```diff
@@ -4261,5 +4261,17 @@
     "telegram": "grahamde8313",
     "operatorName": "Derek Graham",
     "github": "grahamde8313"
+  },
+  {
+    "ticker": "ANTRX",
+    "poolId": "8b315cd9c1b128c5cfe70925a786f6db96cf220bbcea8a2a9c6afdb9",
+    "social": {
+      "website": "",
+      "twitter": "Antrix94841957",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "Antrix",
```

Comment:
Please can you add in the `operatorName` and `github` fields? Even if you leave them blank?

operatorName must be either blank or your real name!